### DISCUSSION TITLE
References and useful stuff

### DISCUSSION THREAD

**Discussion by eeholmes** [UW-Upwelling-Project/Summer-2022#14] [Team Posts]
## Upcoming conference

https://www.ebus-lima2022.com/program

Interesting El Nino dataset
https://archive.ics.uci.edu/ml/datasets/El+Nino

Upwelling systems of the world
https://link.springer.com/book/10.1007/978-3-319-42524-5

**Comment by zhengx25-1976320** [comment]
interesting paper
https://www.nature.com/articles/nature14235

**Comment by eeholmes** [comment]
Paper on upwelling indices good ref on the Bakun upwelling index
https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2018JC014187

**Comment by eeholmes** [comment]
Lau, K. M., and Hengyi Weng. “Climate Signal Detection Using Wavelet Transform: How to Make a Time Series Sing.” Bulletin of the American Meteorological Society, vol. 76, no. 12, 1995, pp. 2391–402. JSTOR, http://www.jstor.org/stable/26232616. Accessed 31 Jul. 2022.

This one is particularly interesting. For more papers, please read the section 1 of this paper.

Torrence, Christopher and Gilbert P. Compo. “A Practical Guide to Wavelet Analysis.” Bulletin of the American Meteorological Society 79 (1998): 61-78.

Gerald Kaiser. 1994. A friendly guide to wavelets. Birkhauser Boston Inc., USA.

For reference of details of wavelets, please refer to this paper.

**Comment by eeholmes** [comment]
upwelling in the artic https://agupubs.onlinelibrary.wiley.com/doi/10.1002/2016JC012624
